src/v2i-hub/PedestrianPlugin/scripts/websocketServer.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
+import websockets
+import json
+
+# Sample Pededertian Presence Tracking Data
```

Comment:
Provide description for the purpose of this script, how the communication flows between the Pedestrian plugin and this websocket server?

Reply:
Added description